.github/workflows/application-signals-e2e-test.yml

```diff
@@ -18,7 +21,14 @@ concurrency:
 
 
 jobs:
+  CheckBuildTestArtifacts:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      - run: echo 'The triggering build workflow succeeded'
```

Comment:
So this job only echos ?

Reply:
The main purpose of this step is to act as a blocker. If the build fails, then the tests shouldn't run. Think a job requires there to be a step.

Reply:
Makes sense